frontend/src/Frontend/Page/Transaction.hs

```diff
@@ -54,77 +57,104 @@ transactionPage
+          el "tr" $ el "td" $ case _pactCommand_payload $ _transaction_cmd t of
+            ExecPayload exec ->  routeLink route $ text $ _exec_code exec
+            ContPayload cont ->  routeLink route $ text $ _cont_pactId cont
+
+transactionPage
+  :: ( MonadApp r t m
+     , RouteToUrl (R FrontendRoute) m
+     , SetRoute t (R FrontendRoute) m
+     , Prerender js t m
+     , Routed t Text m
+     )
+  => NetId
+  -> ChainId
+  -> BlockPayloadWithOutputs
+  -> m ()
+transactionPage netId cid bp = do
+  dynTxHash :: Dynamic t Text <- askRoute
+  let txs = M.fromList $ map (\(t, tout) -> ((T.pack $ show $ _transaction_hash t), (t, tout))) $  _blockPayloadWithOutputs_transactionsWithOutputs bp
+  _ <- switchHold never <=< dyn $ ffor dynTxHash $ \txHashTxt -> do
+    let mTout = M.lookup txHashTxt txs
+    case mTout of
+      Nothing -> do
+        el "p" $ text "Transaction Details not found."
+        return never
+      Just (t, tout) -> do
         let cmd = _transaction_cmd t
-        let addActive cls active =
-              ("class" =: if active then ("active " <> cls) else cls)
-        (e,_) <- elDynAttr' "div" (addActive "title" <$> open) $ do
-          elClass "i" "dropdown icon" blank
-          elClass "pre" "custombreak" $ text $ payloadCode $ _pactCommand_payload cmd
-        elDynAttr "div" (addActive "content" <$> open) $ do
-          elClass "table" "ui definition table" $ do
-            el "tbody" $ do
-              tfield "Request Key" $ do
-                let reqKey = hashB64U $ _transaction_hash t
-                text reqKey
-              tfield "Transaction Output" $ do
-                elClass "table" "ui definition table" $ el "tbody" $ do
-                  tfield "Gas" $ text $ tshow $ _toutGas tout
-                  tfield "Result" $ text $ join either unwrapJSON $ fromPactResult $ _toutResult tout
-                  tfield "Logs" $ text $ maybe "null " hashB64U $ _toutLogs tout
-                  tfield "Metadata" $ renderMetaData netId cid $ _toutMetaData tout
-                  tfield "Continuation" $ voidMaybe renderCont $ _toutContinuation tout
-                  tfield "Transaction ID" $ maybe blank (text . tshow) $  _toutTxId tout
-              tfield "Events" $ elClass "table" "ui definition table" $ el "tbody" $
-                forM_ (_toutEvents tout) $ \ ev -> el "tr" $ do
-                  elClass "td" "two wide" $ text (ename ev)
-                  -- el "td" $ el "pre" $ text $ prettyJSON ev
-                  elClass "td" "evtd" $ elClass "table" "evtable" $
-                    forM_ (params ev) $ \v -> elClass "tr" "evtable" $ elClass "td" "evtable" $ text $ pactValueJSON v
+        elClass "table" "ui definition table" $ do
+          el "tbody" $ do
+            tfield "Request Key" $ do
+              let reqKey = hashB64U $ _transaction_hash t
+              text reqKey
+            tfield "Transaction Output" $ do
```

Comment:
The transaction's code field does not appear anywhere on this page.  We should add it here, right below the request key field and above the transaction output field.